plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -103,7 +102,7 @@ private static String prepareCassandraYaml()
     {
         String original = Resources.toString(getResource("cu-cassandra.yaml"), UTF_8);
 
-        File tempDirFile = createTempDir();
+        File tempDirFile = Files.createTempDirectory("tmp").toFile();
```

Comment:
Please static import `createTempDirectory` everywhere, since the name is self-descriptive and the `Files.` prefix doesn't add any value.